platform/throughput-report/index.md

```diff
@@ -8,15 +8,18 @@ related:
   - servicepulse/usage-config
 ---
 
-The Particular Service Platform is licensed based on the number of NServiceBus endpoints and the message throughput of those endpoints. In order to determine these values, a monitor needs to be run over the transport broker for a representative period of time (24 hours) and a report generated from this data to be sent to Particular.
+The Particular Service Platform is licensed based on the number of NServiceBus endpoints and the message throughput of those endpoints. In order to determine these values, a process needs to be run that queries the transport broker for up to 24 hours and a report generated from this data to be sent to Particular.
 
 The minimal installation required to generate the usage report is:
 
-- [ServiceControl](/servicecontrol/). Only a single [error instance](/servicecontrol/servicecontrol-instances/) is required, as this is the primary service that contains the monitoring logic.
+- [ServiceControl](/servicecontrol/). Only a single [error instance](/servicecontrol/servicecontrol-instances/) is required, as this is the primary service that contains the broker querying logic.
 - [ServicePulse](/servicepulse/). This is the UI that interfaces with the ServiceControl service to allow users to specify which endpoints are NServiceBus related and generate the usage report to send to Particular.
 
 ## Installation options
 
+> [!WARNING]
```

Comment:
I decide to pull all these sections into a single one, and also not mention `SERVICEBUS_ERRORQUEUE` given that is not required to change that at all.

Reply:
Also, it mentioned "alternative" to set `SERVICEBUS_ERRORQUEUE` to a non existing queue, but without setting `SERVICECONTROL_INGESTERRORMESSAGES=false` we would still be running all the queries, which costs money on Cloud environments.
So I don't think it was correct to say "alternative".